modules/lvms-creating-lvms-cluster-using-cli.adoc

```diff
@@ -0,0 +1,151 @@
+= Creating an LVMCluster CR by using the CLI
+
+You can create an `LVMCluster` custom resource (CR) on a worker node using the OpenShift CLI (`oc`).
+
+[IMPORTANT]
+====
+You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to {product-title} as a user with `cluster-admin` privileges.
+
+* You have installed {lvms}.
+
+* You have installed a worker node in the cluster.
+
+* You read the "About the LVMCluster custom resource" section.
+
+.Procedure
+
+. Create an `LVMCluster` custom resource (CR) YAML file:
++
+.Example `LVMCluster` CR YAML file
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+# ...
+  storage:
+    deviceClasses: <1>
+# ...
+      nodeSelector: <2>
+# ...
+      deviceSelector: <3> 
+# ...
+      thinPoolConfig: <4>
+# ...
+----
+<1> Contains the configuration to assign the local storage devices to the LVM volume groups.
+<2> Contains the configuration to choose the nodes on which you want to create the LVM volume group. If this field is empty, all nodes without no-schedule taints are considered.
+<3> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.  
+<4> Contains the configuration to create a thin pool in the LVM volume group.
+
+. Create the `LVMCluster` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name>
+----
++
+.Example output
+[source,terminal]
+----
+lvmcluster/lvmcluster created
+----
+
+.Verification
+
+. Check that the `LVMCluster` CR is in the `Ready` state:
++
+[source, terminal]
+----
+$ oc get lvmclusters.lvm.topolvm.io -o jsonpath='{.items[*].status.state}' -n <namespace>
+----
++
+.Example output
+[source,json]
+----
+{"deviceClassStatuses": <1>
+[ 
+  {
+    "name": "vg1", 
+    "nodeStatus": [ <2>
+        {
+            "devices": [ <3>
+                "/dev/nvme0n1",
+                "/dev/nvme1n1",
+                "/dev/nvme2n1"
+            ],
+            "node": "kube-node", <4>
+            "status": "Ready" <5>
+        }
+    ]
+  }
+]
+"state":"Ready"} <6>
+----
+<1> The status of the device class. 
+<2> The status of the LVM volume group on each node.
+<3> The list of devices used to create the LVM volume group.
+<4> The node on which the device class is created.
+<5> The status of the LVM volume group on the node.
+<6> The status of the `LVMCluster` CR.
++
+[NOTE]
+====
+If the `LVMCluster` CR is in the `Failed` state, you can view the reason for failure in the `status` field.
+
+Example of `status` field with the reason for failue:
+[source, yaml]
+----
+status:
+  deviceClassStatuses:
+    - name: vg1
+      nodeStatus:
+        - node: my-node-1.example.com 
+          reason: no available devices found for volume group
+          status: Failed
+  state: Failed
+----
```

Comment:
There's a typo ("failue"), but also I feel like this sentence could just be the title for the block instead?
```suggestion
.Example `status` field with the reason for failure
[source,yaml]
----
status:
  deviceClassStatuses:
    - name: vg1
      nodeStatus:
        - node: my-node-1.example.com 
          reason: no available devices found for volume group
          status: Failed
  state: Failed
----
```

Reply:
Updated as per the suggestion.